.cirrus.star

```diff
@@ -26,10 +26,10 @@ def main(ctx):
     response = http.get(url).json()
     commit_msg = response["message"]
 
-    if "[skip ci]" in commit_msg:
-        return []
-
     if "[cd build]" in commit_msg or "[cd build cirrus]" in commit_msg:
-        return fs.read(arm_wheel_yaml) + fs.read(arm_tests_yaml)
+        return fs.read(arm_wheel_yaml)
```

Comment:
Before running tests and building wheels was done together.

Are tests run when the wheels are built? If not, could we change the logic so that we can trigger both when several tags are present?

Reply:
I updated the PR to run the required jobs based on the tags. This means that if both `[cd build]` and `[cirrus arm]` is present, then both jobs run.

Reply:
Thanks!